src/tools/info/sys_info.c

```diff
@@ -176,7 +191,33 @@ static void print_sys_topo()
         print_row_separator(distance_width, name_width, num_devices, ' ', '|');
         print_row_separator(distance_width, name_width, num_devices, '-', '+');
     }
-    printf("#\n");
+
+    print_table_header(memory_latency_title, numa_distance_unit, distance_width,
+                       name_width, num_devices);
+    print_row_separator(distance_width, name_width, num_devices, ' ', '|');
+
+    printf("# |%*s ", name_width - 1, "");
+    printf("|");
+    for (sys_dev1 = 0; sys_dev1 < num_devices; ++sys_dev1) {
+        status = ucs_topo_get_memory_distance(sys_dev1, &distance);
+        if (status != UCS_OK) {
+            ucs_snprintf_safe(distance_str, sizeof(distance_str), "<error %d>",
+                              status);
+        } else {
+            ucs_snprintf_safe(distance_str, sizeof(distance_str), "%.1f",
+                              distance.latency * UCS_NSEC_PER_SEC);
+        }
+        printf("%*s |", distance_width - 1, distance_str);
+    }
+
+    printf("\n");
+    print_row_separator(distance_width, name_width, num_devices, ' ', '|');
+    print_row_separator(distance_width, name_width, num_devices, '-', '+');
```

Comment:
can you give example of the output?

Reply:
https://github.com/openucx/ucx/pull/8853#discussion_r1144990075

Reply:
can you pls fix units/dev name?
see https://github.com/openucx/ucx/pull/8853#discussion_r1167891093